Yearn-agents/InstantWithdrawl/src/blockchain.ts

```diff
@@ -0,0 +1,32 @@
+import path from "path";
+import dotenv from "dotenv";
+const ganache = require("ganache-cli");
```

Comment:
use `import ganache from "ganache-core"`

Reply:
The `ganache-core` is last updated 9 months ago. The latest updated one is `@ganache/core`. For that this is what I am getting:
![image](https://user-images.githubusercontent.com/43861614/142873631-5a59b515-3cc7-4c73-990f-c374952d410b.png)
So hence I used the `ganache-cli`